examples/seq2seq/seq2seq.py

```diff
@@ -0,0 +1,173 @@
+import argparse
+import collections
+
+from nltk.corpus import comtrans
+from nltk.translate import bleu_score
```

Comment:
Yes, I'll write README for this example in this directory. Other examples also uses additional libraries.